runtime-transaction/src/runtime_transaction.rs

```diff
@@ -27,7 +28,6 @@ use {
     std::collections::HashSet,
 };
 
-#[cfg_attr(test, derive(Eq, PartialEq))]
```

Comment:
looks like tests somewhere no longer does comparison? Anyway, less is more

Reply:
things compiled when I removed it. I originally removed it because it appears that using `#[cfg_attr(test, ...)]` doesn't work correctly across different crates.

Didn't want to unconditionally add these derives to the sdk type.